maquetado/.gitignore

```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```

Comment:
Remember to add an empty line at the end of the file. It helps text processors to ease the reading and parsing of text files. [Here's](https://thoughtbot.com/blog/no-newline-at-end-of-file) some info about it.